experimental/PropertyDDS/packages/property-dds/src/test/propertyTree.spec.ts

```diff
@@ -246,6 +246,15 @@ describe("PropertyTree", () => {
 
 				expect(count).to.equal(1);
 			});
+
+            it("getRebasedChanges should return empty array empty guid as start & end", async () => {
```

Comment:
Nit: In the test title, did you mean '...return empty array as...'  vs '...return empty array empty guid as...'?

(If so, fixing in your next PR is fine...)